src/app/contextBroker/contextBroker.cpp

```diff
@@ -185,6 +185,7 @@
 #include "serviceRoutinesV2/getEntityAllTypes.h"
```

Comment:
CNR line missing (I could add it in the PR implementing this at mongoBackend, but better to add as soon as we start working on the item)